packages/now-cli/test/dev/integration.js

```diff
@@ -435,6 +435,15 @@ test(
   })
 );
 
+test(
+  '[now dev] should preserve query string even after miss phase',
+  testFixtureStdio('handle-miss-querystring', async testPath => {
+    await testPath(200, '/', 'Index Page');
+    await testPath(200, '/echo/first/second', 'a=first,b=second');
```

Comment:
This test doesn't seem to be testing anything other than the first route

Reply:
The first route has a destination of `/functions/echo.js?a=$1&b=$2` (notice the extension `.js`). But the file is renamed to the extension-less `/functions/echo` so that's why it was failing before because the query string was not preserved.

Reply:
Oh ok